contracts/HubPool.sol

```diff
@@ -398,17 +418,16 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
      *              INTERNAL FUNCTIONS               *
      *************************************************/
 
-    function _exchangeRateCurrent() internal pure returns (uint256) {
-        // TODO: implement this method to consider utilization.
-        return 1e18;
-    }
-
     // Unwraps ETH and does a transfer to a recipient address. If the recipient is a smart contract then sends WETH.
-    function _unwrapWETHTo(address payable to, uint256 amount) internal {
+    function _unwrapWETHTo(
```

Comment:
this contract no longer stores a `l1Weth` instance. Rather, the `pooledTokens` flags `weth` with a `isWeth` bool, which is passed in here.